### DISCUSSION TITLE
How do I ignore rules for an entire folder/workspace?

### DISCUSSION THREAD

**Discussion by Prodeguerriero** [davidlday/vscode-languagetool-linter#742] [Q&A] [answered]
Hi all,

I must be missing something obvious, but while I can find the settings to manage ignored _words_, I can't seem to find the same to disable rules altogether.

Do I only have [the option to ignore inline](https://github.com/davidlday/vscode-languagetool-linter?tab=readme-ov-file#ignore-rules-inline), or is there something else that is escaping me?

Thanks

**Comment by davidlday** [comment, marked as answer]
Hey! Sorry for not seeing this sooner. There is an option to ignore in the workspace. All extensions have this feature for having settings specific to a workspace, but I think how I had to handle ignored words may be confusing how VSCode's settings work. If you go to settings, you'll see at the top (at least) two option: User and Workspace. My screenshot has three b/c I use WSL on my Windows machine.

![image](https://github.com/user-attachments/assets/68604819-84a0-41d2-bc90-f6c8c742e199)

If you select "Workspace" just under the search settings, those will apply only to the workspace you're in. Hope this helps!

**Reply by Prodeguerriero** [reply]
Thanks. I have also found the contextual option by right-clicking on the error message. I could swear I had tried this before opening this issue, but apparently I didn't (or I had a brain fart).

Thanks.

**Reply by Prodeguerriero** [reply]
Actually, quick follow-up question @davidlday 

Is there a way to use the inline ignore command to be applied to an entire file? Markdown lint [does something similar](https://github.com/DavidAnson/markdownlint?tab=readme-ov-file#configuration).

Cheers,

**Reply by davidlday** [reply]
@Prodeguerriero - I'll have to look again at the inline ignore options. That was a contribution, so I don't know off the top of my head, but I don't believe it currently applies to the whole file. Feel free to open a feature request, but I'll look into in the meantime.
